tests/unit/aiplatform/test_automl_forecasting_training_jobs.py

```diff
@@ -489,16 +513,24 @@ def test_run_call_pipeline_if_no_model_display_name_nor_model_labels(
 
     @pytest.mark.usefixtures("mock_pipeline_service_get")
     @pytest.mark.parametrize("sync", [True, False])
+    @pytest.mark.parametrize(
```

Comment:
training_jobs.AutoMLForecastingTrainingJob and training_jobs.SequenceToSequencePlusForecastingTrainingJob will not necessarily share the same API forever. Prefer splitting into two separate tests.

Reply:
I'm torn. I'm also open to deferring this until it's needed. Feel free to decide.

Reply:
Let's leave it as-is for now. The intention is for Seq2Seq to share the same API as AutoML, but we're leaving the door open for implementation differences on the SDK side by splitting into two classes instead of just having a `model_type` argument in the `AutoMLForecastingTrainingJob`. At least that's my understanding.

Reply:
Sure, we can revisit later. Thanks.